pandas/core/series.py

```diff
@@ -1032,6 +1042,34 @@ def reset_index(self, level=None, drop=False, name=None, inplace=False):
         2   c  3
         3   d  4
 
+        >>> s = pd.Series([1, 2, 3, 4], index=pd.Index(['a', 'b', 'c', 'd'],
+        ...                                            name = 'idx'))
+        >>> s.reset_index(drop=True)
+        0    1
+        1    2
+        2    3
+        3    4
+        dtype: int64
+
+        >>> s = pd.Series([1, 2, 3, 4], index=pd.Index(['a', 'b', 'c', 'd'],
+        ...                                            name = 'idx'))
+        >>> s.reset_index(name = 'new_idx')
+          idx  new_idx
+        0   a        1
+        1   b        2
+        2   c        3
+        3   d        4
+
+        >>> s = pd.Series([1, 2, 3, 4], index=pd.Index(['a', 'b', 'c', 'd'],
```

Comment:
you don't need to re-define the series, you can reuse from the previous code block

Reply:
Can you also put some explanation sentence between the examples?